sailthru_import/test/test_sailthru_import.py

```diff
@@ -0,0 +1,14 @@
+from sailthru_import import sailthru_import
+from sailthru.sailthru_client import SailthruClient
+
+import unittest
+import mock
+from mock import patch
+
+
+@patch('sailthru.sailthru_client')
+class SailThruTestCase(unittest.TestCase):
+    def test_upload_csv_to_sailthru(self, mock_client):
+        mock_client.return_value.api_get.response.body.status.return_value = "completed"
+        mock_client.upload_csv_to_sailthru('test.csv', 'list name', 'bfohl@edx.org', mock_client)
+        mock_client.upload_csv_to_sailthru.assert_called_once()
```

Comment:
As we talked about: use `sailthru_import.upload_csv_to_sailthru`, and then `mock_client.api_post.assert_called_once`.

Also, `assert_called_once_with(...)` will allow you to validate the arguments that are passed to sailthru.